Starcounter.Linq.Tests/FirstOrDefaultTests.cs

```diff
@@ -21,6 +21,20 @@ public void ValueEquals()
         }
 
         [Fact]
+        public void ValueNotEquals()
+        {
+            Assert.Equal("SELECT P FROM Starcounter.Linq.Tests.Person P WHERE ((NOT P.Name = ?)) FETCH 1",
```

Comment:
You can use `<>` operator instead of the `NOT` hack.

Reply:
Done